hetmech/test_matrix.py

```diff
@@ -1,9 +1,53 @@
 import hetio.readwrite
+import numpy
+import pytest
+from scipy import sparse
+
 from .matrix import metaedge_to_adjacency_matrix
-import numpy as np
 
 
-def test_metaedge_to_adjacency_matrix():
+def get_arrays(edge, mat_type, dtype):  # Define a dictionary
+    rows_dict = {
```

Comment:
Not certain I understand. They contain the same **set** of values, but `rows_dict['GaD']` != `cols_dict['GaD']`.